### PR TITLE
Update path to sample data

### PR DESCRIPTION
**Pull request by torbsorb**

### CODE REVIEW

**Comment by SatjaSivcev** [comment]
Can you review this one as well @eskaur ?

File: README.md [hunk 1]
```diff
@@ -66,7 +66,13 @@ Note: The version tested with Zivid cameras is 1.8.1.
 
        pip install -r requirements.txt
 
-6. Open and run one of the samples.
+6. Add the directory `source` to PYTHONPATH. Navigate to the root of the repository and run:
+
+    - PowerShell: `$env:PYTHONPATH=$env:PYTHONPATH + ";$PWD\source"`
+    - cmd: `set PYTHONPATH="$PYTHONPATH;$PWD\source"`
+    - bash: `export PYTHONPATH="$PYTHONPATH:$PWD/source"`
+
```
Comment (eskaur) [inline README.md:74]: Why is this necessary?

Comment (eskaur) [reply README.md:74]: Oh, to get access to get_sample_data_path :)

**Review by eskaur — APPROVED** [review]
